docs/autosetup-regtest-dao/README.md

```diff
@@ -0,0 +1,188 @@
+
+We assume the user is in the `autosetup-regtest-dao` directory. Is it ok to move the directory, but please move it as a whole. Also `bitcoin-qt`, `bitcoin-cli`, `bitcoin-tx`, `jq` and `bc` has to be installed on the system. Do not change the shipped `bitcoin.conf` file as it will break things.
+
+## Steps to spin up a fresh DAO (we assume the user is in the autosetup-regtest-dao directory):
+1. Set up `bitcoin-qt` to run in the `autosetup-regtest-dao` directory, it detects when the `bitcoin-qt` instance comes up and once it is ready the script queries the number of blocks. If there are less then 101 blocks (fresh start) the script generates 101 blocks for you so you have an available input that will be later used for the genesis tx creation.
+```
+./start_bitcoind.sh
+```
+Example output (note the below errors are normal and can be ignored):
+```
+user@host:~/bin/KanoczTomas/bisq/docs/autosetup-regtest-dao$ ./start_bitcoind.sh 
+error: Could not connect to the server 127.0.0.1:18443
+
+Make sure the bitcoind server is running and that you are connecting to the correct RPC port.
+error code: -28
+error message:
+Loading wallet...
+./generate_101_blocks.sh: found less blocks then 101, generating ...
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+./blocknotify: line 2: echo: write error: Broken pipe
+[
+  "24e04970e3b55fe45efde314cc959a7baeef379d74d322bc132b723235f1e8ca",
+  <truncated output for compactness>
+  "2d4aaa455c681b604a683437c657e23b19af0b614f5f822c4c7aecee865b5f5f"
+]
+./generate_101_blocks.sh: done, exiting
+```
+2. Open a new terminal (preferably a new tab), go to the `bisq` folder and start the `create_genesis.sh` script. The script tests for a running bitcoind node in regtest, so make sure step 1 was successful. 
+```
+cd bisq
+./create_genesis.sh
+```
+example output:
+```
+user@host:~/bin/KanoczTomas/bisq/docs/autosetup-regtest-dao/bisq$ ./create_genesis.sh 
+testing if bitcoind is running: bitcoind is running, all is ok!
+Please provide Alice's BSQ Address
+
+```
+3. We need the BSQ address of `Alice`. We need to go to the bisq directory and start the `Alice` instance. Wait until the instance starts up then go to `DAO -> Receive` and copy the BSQ address. Note the address starts with a `B`. Open a new terminal and enter the below commands to start `Alice`. Then wait for the gui to start and paste the BSQ address to `create_genesis.sh` which is still running from step 2.
```

Comment:
I think `..enter the below commands..` should be `... enter the commands below..` , but I'm not a native speaker myself ;-)

Reply:
Yes you are right ...